packer.py

```diff
@@ -48,24 +52,26 @@ def create_bins(bins: dict) -> list:
             bins[bin]['max_weight']))
     return bins_list
 
+
 def refresh_items(unfitted_items, packer):
     for item in unfitted_items:
         packer.add_item(item)
 
+
 def refresh_bin_types(bin_types, packer):
     for bin in bin_types:
         packer.add_bin(bin)
 
-def execute_packing(items_to_fit: list, bin_types: list, visualize=True, export_img=False, textualize=True) -> list:
+
+def execute_packing(items_to_fit: list, bin_types: list, visualize=False, export_img=False, textualize=False) -> list:
     fitted_items = []   # list of solutions
     tree = Tree("Packing list:", highlight=True, hide_root=True)
     while items_to_fit:
-        packing_efficacy = 0
         packer = Packer()
         refresh_bin_types(bin_types, packer)
         refresh_items(items_to_fit, packer)
         packer.pack(bigger_first=True)
-        best_bin = max(packer.bins, key=lambda b: b.efficacy)    # select the best packed bin
+        best_bin = deepcopy(max(packer.bins, key=lambda b: b.efficacy)) # select the best packed bin and copy it
```

Comment:
Why deepcopy ?

Reply:
My idea of the results being returned by `execute_packing()` function is to receive a tuple `(Bins objects, Items objects packed in each of those Bins` (conserving Items position and rotation in each of the Bins). So that in the end we can know how many bins were used, what are their types and what Items are inside each of the Bin, and how were they packed there.

In order to receive it I decided to separate (by deepcoping) the resulting Bin and Items objects from the ones that are inside the Packer instance. Otherwise (if we don't deepcopy) - the Bin in the results will always refer to one of the bins loaded to Packer (which are the types of bins, not the "real" bins) and the Items position and rotation will be changed with each iteration of the while function.